lfs/setup.go

```diff
@@ -26,8 +26,7 @@ var (
 	filters = &Attribute{
 		Section: "filter.lfs",
 		Properties: map[string]string{
```

Comment:
I wonder if we'll have to support a way to "upgrade" these (similar to how we treat the hooks in the `lfs` package, according to https://github.com/github/git-lfs/pull/1083)